cmapPy/math/modz.py

```diff
@@ -0,0 +1,68 @@
+'''
+modz.py
+
+Given a matrix of profiles on which you want to calculate a weighted average, returns a single profile of modz values
+'''
```

Comment:
Would be good to describe how the weighted average is computed. That is, "Replicate profiles are weighted based on their correlation with each other. If one replicate is an outlier, its values will be down-weighted relative to the values of the other replicate profiles."

Reply:
Would also be good to describe up here that there are some tweak-able parameters like `min_wt` and `corr_metric.` And that negative similarities are converted to 0s.

Reply:
Also, since this is publicly available, "modz" needs to be further defined.

Reply:
The very first comment of this script is basically "hey, we know this name isn't really correct." Maybe it's worth naming it the right thing while we have the chance?

`modz` aggregates a matrix of replicate profiles. And it only works for a matrix in which all the replicates belong to the same perturbagen; in other words, some other wrapper needs to apply `modz` to all replicate sets in a larger matrix. So what if `modz` became `agg_one`, and the script we later add will be `agg_replicates`?